azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerTest.java

```diff
@@ -69,9 +77,11 @@ public void setUp() throws Exception {
     jobtypeManager =
         new JobTypeManager(null, null, this.getClass().getClassLoader());
     JobTypePluginSet pluginSet = jobtypeManager.getJobTypePluginSet();
-    pluginSet.addPluginClass("java", JavaJob.class);
+    pluginSet.setCommonPluginLoadProps(AllJobExecutorTests.setUpCommonProps());
     pluginSet.addPluginClass("test", InteractiveTestJob.class);
     fakeProjectLoader = new MockProjectLoader(workingDir);
+    Utils.initServiceProvider();
+    AzkabanExecutorServer.setApp(new FakeApp());
```

Comment:
The refactoring of ExecSever class is useful.
Is it possible to use a mock class instead in these tests so that we can separate that refactoring change into a separate pull request and finish this one first?